pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTaskRestletResource.java

```diff
@@ -346,6 +347,29 @@ public StringResultResponse getTaskStateDeprecated(
     }
   }
 
+  @GET
+  @Path("/tasks/all")
```

Comment:
when this API would be used? I see `/tasks/{taskType}/taskstates` is there.

Reply:
We don't have one API to show all the task states.

Reply:
Can we do that in the client? (i.e. fetch all tasks and then iterate through the types to get status)?

Reply:
The goal eventually is to convert those API responses to an in-memory table and you can run the below queries to see job status.

```
SHOW LOAD;
SHOW LOAD WHERE taskName = 'mytask-01';
SHOW LOAD WHERE state = 'CANCELLED';
SHOW LOAD WHERE state = 'LOADING';
```

I feel we may need a better data structure to hold the task status, e.g. job progress(12/35 COMPLETED, 5/35 IN_PROGRESS ), exception: xxxxx, time usage: 12mins, etc.

Reply:
I will remove this API for now, we can add it later on with a better understanding of the task information.